met-web/src/components/engagement/admin/create/authoring/AuthoringFeedback.tsx

```diff
@@ -0,0 +1,281 @@
+        height: '48px',
+        marginTop: '8px',
+        padding: '0',
+    };
+    const toolbar = {
+        options: ['inline', 'list', 'link', 'blockType', 'history'],
+        inline: {
+            options: ['bold', 'italic', 'underline', 'superscript', 'subscript'],
+        },
+        blockType: { options: ['Normal', 'H2', 'H3', 'Blockquote'] },
+        list: { options: ['unordered', 'ordered'] },
+    };
+
+    const handleEditorChange = (newEditorState: EditorState) => {
+        const plainText = newEditorState.getCurrentContent().getPlainText();
+        setEditorState(newEditorState);
+        setBodyText(plainText);
+    };
+
+    const handleWidgetChange = (event: SelectChangeEvent<string>) => {
+        setCurrentWidget(event.target.value);
+    };
+
+    const handleRemoveWidget = () => {
+        if ('' === currentWidget) {
+            return;
+        } else {
+            setCurrentWidget('');
+        }
+    };
+
+    return (
+        <Grid container sx={{ maxWidth: '700px', mt: '1rem' }} direction="column">
+            <Grid item sx={{ mt: '1rem' }}>
+                <MetHeader3 style={metHeader3Styles}>Primary Content (Required)</MetHeader3>
+                <EyebrowText style={eyebrowTextStyles}>
+                    This section of content should provide a brief overview of what your engagement is about and what
+                    you would like your audience to do.
+                </EyebrowText>
+            </Grid>
+
+            <Grid sx={{ ...formItemContainerStyles, backgroundColor: colors.surface.blue[10] }} item>
+                <label htmlFor="section_heading">
+                    <MetHeader3 style={metHeader3Styles}>
+                        Section Heading <span style={{ fontWeight: 'normal' }}>(Required)</span>
+                    </MetHeader3>
+                    <EyebrowText style={eyebrowTextStyles}>
+                        Your section heading should be descriptive, short and succinct.
+                    </EyebrowText>
+                    <TextField
+                        sx={{ backgroundColor: colors.surface.white }}
+                        value={sectionHeading}
+                        id="section_heading"
+                        counter
+                        maxLength={60}
+                        placeholder="Section heading message"
+                        onChange={(value) => {
+                            setSectionHeading(value);
+                        }}
+                    />
+                </label>
+            </Grid>
+
+            <Grid sx={{ ...formItemContainerStyles, backgroundColor: colors.surface.gray[10] }} item>
+                <label htmlFor="body_copy">
+                    <MetHeader3 style={metHeader3Styles}>
+                        Body Copy
+                        <span style={{ fontWeight: 'normal' }}> (Required)</span>
+                    </MetHeader3>
+                    <EyebrowText style={eyebrowTextStyles}>
```

Comment:
We're giving sighted users all this nice information about what goes in this field. Is it easy for screen reader users to perceive this as well?

Reply:
That's a good question! It's harder to test screen reader compatibility through Firefox without ChromeVox, but I found a plugin called Pericles and tested it with the page. The text was read by the screenreader, so we should be ok.

Reply:
Thanks for testing that! It's important not only that a screen reader can read it but also that it's clear that this information is associated with a particular form field. When you tabbed your focus to this field, did you hear all this text read out?

Reply:
The tab just cycles through links, I believe in ChromeVox to cycle through individual items it's something like CTRL+ALT+CMD+DOWN ARROW (or up arrow). The screen reader reads the entire page including this text otherwise.